Extensions/people_notifier.js

```diff
@@ -360,6 +366,45 @@ XKit.extensions.people_notifier = new Object({
 			$(this).find('.count').css('marginRight', '0');
 		});
 
+		var dragging;
+		var dragtarget;
+		var mouseY;
+		var half_div_height = 13;
+		$(document).on("dragstart", ".xkit-people-notifier-person", function(e){
+			dragging = $(this);
+			e.originalEvent.dataTransfer.dropEffect = "move";
+		});
+		$(document).on("drag", ".xkit-people-notifier-person", function(e){
+			mouseY = e.originalEvent.pageY;
+		});
+		$(document).on("dragenter", ".xkit-people-notifier-person", function(e){
+			e.preventDefault();
+			dragtarget = $(this);
+		});
+		$(document).on("dragover", ".xkit-people-notifier-person", function(e){
+			e.preventDefault();
+		});
+		$(document).on("drop", ".xkit-people-notifier-person", function(e){
+			if (dragtarget[0] !== dragging[0]) {
+				var target_index = XKit.extensions.people_notifier.blogs.indexOf(XKit.extensions.people_notifier.searchable_blogs[$(dragtarget).attr('data-url')]);
+				var dragging_obj = XKit.extensions.people_notifier.searchable_blogs[$(dragging).attr('data-url')];
+				var dragging_index = XKit.extensions.people_notifier.blogs.indexOf(dragging_obj);
+				var top_of_target = dragtarget.offset().top;
+				dragging.detach();
+				if ((mouseY - top_of_target) > half_div_height) {
+					dragtarget.after(dragging);
+				} else {
+					dragtarget.before(dragging);
+					target_index--;
```

Comment:
If you try to drag a blog to the top-most position, `target_index` will be decremented to `-1`, causing the extension to call `XKit.extensions.people_notifier.blogs.splice(-1, 0, dragging_obj);` and place it in the second to last position.

https://gfycat.com/DefiniteSlimyAustraliansilkyterrier